res/css/views/messages/_ViewSourceEvent.scss

```diff
@@ -19,8 +19,10 @@ limitations under the License.
     opacity: 0.6;
     font-size: $font-12px;
     width: 100%;
+    overflow-x: auto; // Cancel overflow setting of .mx_EventTile_content
```

Comment:
I think `scroll` might make sense here?

Reply:
`scroll` creates scrollbar space whether or not the content is actually clipped. I think it is not expected here.

https://developer.mozilla.org/en-US/docs/Web/CSS/overflow-x

Reply:
Ah, NVM, I got confused